### PR TITLE
add a security page

### PR DESCRIPTION
**Pull request by ru-fu**
Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: exp/security.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+|--------------------------------------------|:------------------------:|
+| App sandbox                                | ✓                        |
+| App signing                                | ✓                        |
+| Authentication                             | ?                        |
+| Biometrics                                 | ?                        |
+| Encryption                                 | -                        |
+| Keystore                                   | ?                        |
+| Security-Enhanced Linux (SELinux)          | -                        |
+| Trusty Trusted Execution Environment (TEE) | -                        |
+| Verified Boot                              | -                        |
```
Comment (ru-fu) [inline exp/security.md:70]: Need to check and fill the question marks (or actually, the full table ;) ).

**Review by morphis — CHANGES_REQUESTED** [review]

File: exp/security.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+See [GPU slots](https://discourse.ubuntu.com/t/about-capacity-planning/28717#gpu-slots) for more information.
+[/note]
+
+### Security updates
+
+Anbox Cloud provides container images that are frequently updated with the latest security patches. When an image is updated, all Anbox Cloud applications that use the image are automatically updated as well (unless disabled with [`application.auto_update`](https://discourse.ubuntu.com/t/ams-configuration/20872)).
+
+In addition, to ensure that the latest Ubuntu security patches are applied outside of image updates as well, Anbox Cloud checks for and installs available security updates every time an application is bootstrapped. That means that when you create an application, its underlying image is updated with the latest Ubuntu security patches. It is possible to turn off this update mechanism by setting [`container.security_updates`](https://discourse.ubuntu.com/t/ams-configuration/20872) to `false`, but it is not recommended to do so.
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply exp/security.md:50]: Please check again, not 100% sure I understood your comment correctly. ;)

File: exp/security.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Anbox Cloud uses LXD in a way that enforces the highest security level.
+
+### Unprivileged containers
+
+Many container managers use privileged containers, which means that the containers have root privileges on the host system, including access to all devices. This is a security risk, because attackers could gain control over the host system.
+
+Anbox Cloud uses unprivileged LXD containers only, which fully isolates the containers and ensures that they cannot gain root privileges. In addition, the Android container that runs inside the LXD container also runs as an unprivileged container. This method isolates the Android container twice, with the result that if the encapsulation of either the LXD container or the Android container should fail, the system would still be protected.
+
+[note type="information" status="Important"]
+While containers are fully isolated, all containers currently use the same GPU resources. As a result, a container could take all GPU resources in a DDoS-like attack, which would prevent other containers from starting.
```
[the remark this answers is not in the document]
Comment (morphis) [reply exp/security.md]: Alright, lets leave that out then

**Review by morphis — APPROVED** [review]
LGTM, thanks!

**Pull request merged**